input/full.go

```diff
@@ -29,6 +29,24 @@ import (
 func CollectFull(ctx context.Context, server *state.Server, connection *sql.DB, opts state.CollectionOpts, logger *util.Logger) (ps state.PersistedState, ts state.TransientState, err error) {
 	ps.CollectedAt = time.Now()
 
+	bufferCacheReady := make(chan state.BufferCache)
+	go func() {
+		if server.Config.MaxBufferCacheMonitoringGB > 0 {
```

Comment:
This is technically a new condition, but I think its okay to add, and avoids an unnecessary database connection if buffer cache collection is turned off.